testdata/http_multipart.yml.runbook.golden

```diff
@@ -0,0 +1,43 @@
+-- -testdata-book-http_multipart.yml --
+desc: Captured of http_multipart.yml run
+runners:
+  req: '[THIS IS HTTP RUNNER]'
+steps:
+- req:
+    /multipartUpload:
+      post:
+        body:
+          multipart/form-data; boundary=123456789012345678901234567890abcdefghijklmnopqrstuvwxyz: !!binary |
```

Comment:
fmm... There seems to be a bug in https://github.com/k1LoW/runn/blob/main/capture/runbook.go.
Content-Type header is not being retrieved.

Reply:
I'll try to fix this bug with another PR on my part.

Reply:
Ah, Content-Type `multipart/form-data; boundary=123456789012345678901234567890abcdefghijklmnopqrstuvwxyz:` is getting it.